Makefile

```diff
@@ -27,6 +27,10 @@ EXTENSION = .c
 m8c: $(OBJ)
 	$(CC) -o $@ $^ $(local_CFLAGS) $(INCLUDES)
 
+libusb: INCLUDES = $(shell pkg-config --libs sdl2 libusb)
+libusb: local_CFLAGS = $(CFLAGS) $(shell pkg-config --cflags sdl2 libusb) -Wall -O2 -pipe -I.
```

Comment:
This probably needs -DUSE_LIBUSB somewhere to get the libusb code actually baked in :)